pkg/internal/apis/certmanager/validation/certificate.go

```diff
@@ -52,6 +52,15 @@ func ValidateCertificateSpec(crt *cmapi.CertificateSpec, fldPath *field.Path) fi
 	if len(crt.IPAddresses) > 0 {
 		el = append(el, validateIPAddresses(crt, fldPath)...)
 	}
+
+	if len(crt.EmailSANs) > 0 {
+		el = append(el, validateEmailAddresses(crt, fldPath)...)
+	}
+
+	if crt.KeySize < 0 {
```

Comment:
Looks like this change may have snuck back in after rebasing? I think this is now covered by https://github.com/jetstack/cert-manager/pull/2593 in OpenAPI.